ci-operator/step-registry/aws/deprovision/iam-profile/aws-deprovision-iam-profile-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: aws-deprovision-iam-profile
+  from_image:
+    namespace: ocp
+    name: "4.12"
```

Comment:
Can/should we use a more updated image?

Reply:
I guess we can but I wanted to keep it consistent with the iam-role step.

Reply:
The only tool we require in this image is `yq-go` (and `aws`), so 4.12 is good for us, we can use an updated image for both byo-role/profile, but it's not necessary.